crates/bevy_reflect/bevy_reflect_derive/src/derive_data.rs

```diff
@@ -35,7 +35,7 @@ pub(crate) enum ReflectDerive<'a> {
 /// //                          traits
 /// //        |----------------------------------------|
 /// #[reflect(PartialEq, Serialize, Deserialize, Default)]
-/// //            type_name       generics
+/// //            type_path       generics
```

Comment:
i suppose it could be seen as either, but isn't the graph supposed to show what the fields of `ReflectMeta` represent, and there isn't a `type_name` field.